debian/vendor/github.com/blang/semver/README.md

```diff
@@ -1,7 +1,7 @@
 semver for golang [![Build Status](https://travis-ci.org/blang/semver.svg?branch=master)](https://travis-ci.org/blang/semver) [![GoDoc](https://godoc.org/github.com/blang/semver?status.png)](https://godoc.org/github.com/blang/semver) [![Coverage Status](https://img.shields.io/coveralls/blang/semver.svg)](https://coveralls.io/r/blang/semver?branch=master)
 ======
 
-semver is a [Semantic Versioning](http://semver.org/) library written in golang. It fully covers spec version `2.0.0`.
+semver is a [Semantic Versioning] library written in golang. It fully covers spec version `2.0.0`.
```

Comment:
I don't understand this change. http://semver.org is a valid URL (though it can be updated to https:// too.)